public/components/overview/mitre/components/techniques/components/flyout-technique/flyout-technique.tsx

```diff
@@ -130,36 +131,38 @@ export class FlyoutTechnique extends Component {
     }
   }
 
+  findTacticName(tactics){
+    const { tacticsObject } = this.props;
+    const tacticsObj = []
+    tactics.forEach(element => {
+      const tactic = Object.keys(tacticsObject).map(tacticsKey => tacticsObject[tacticsKey]).find(obj => obj.id === element)
```

Comment:
if you need values of an object you should use `Object.values` instead of `Object.keys` and `map`